### ISSUE TITLE
llamacpp model has stopped working somewhere between v0.0.344 & v0.0.355

### ISSUE THREAD

**Issue by changeling** [continuedev/continue#427]
(Using `llamacpp` native server)

After updating to v0.0.350 from v0.0.344, llamacpp has stopped working again. I'm getting:

```
Traceback (most recent call last):

  File "continuedev/src/continuedev/libs/util/create_async_task.py", line 21, in callback
    future.result()

  File "asyncio/futures.py", line 203, in result

  File "asyncio/tasks.py", line 267, in __step

  File "continuedev/src/continuedev/server/gui.py", line 266, in async_stuff
    models.default = MODEL_CLASSES[model_class](**model)
                     ~~~~~~~~~~~~~^^^^^^^^^^^^^

KeyError: 'LlamaCpp'
```

In addition, adding a customization to `config.py` seems to be overwritten when I choose llamacpp.

For the record, v0.0.344 was working fine. I was using it when I saw that there was an update, so I installed v0.0.350, and immediately lost llamacpp.

**Comment by sestinj** [comment]
Must be a merge gone wrong somewhere? Easy fix, I'll push a version in the next hour

**Comment by sestinj** [comment]
Fixed now in 0.0.351

**Comment by sestinj** [comment]
Closing this issue, pretty sure it has been resolved. of course reopen if not
